docs/source/index.rst

```diff
@@ -14,6 +14,7 @@ Here you will find all the relevant information about the project.
    Getting Started <getting_started>
    Libraries <libraries>
    Tutorials <tutorials/tutorials>
+   Guides <guides/guides>
```

Comment:
Maybe this can be above Tutorials? I can also imagine this section under getting started.